guides/two-way-peg-app/advanced-operations.md

```diff
@@ -7,7 +7,171 @@ tags: 2 way peg, powpeg, peg-in, peg-out, 2way-peg, bridge, rbtc, btc, testnet,
+Convert [BTC - RBTC](#converting-btc-to-rbtc) and import a [key in Electrum](#import-key-in-electrum)
+- Selecting [different accounts](#account-selection)
+- Viewing [advanced details](#how-to-view-advanced-details)
+- Adjusting [network fees](#adjusting-network-fees)
+- Viewing [Advanced data](#advanced-data)
+
+## How to view a derived address
+
+Here, we will learn how to view a derived address and use [Liquality](#using-liquality) and [Metamask](#using-metamask) to get a private key. We will also learn how to [convert BTC - RBTC](#converting-btc-to-rbtc) and [Import a Key in Elecrum](#import-key-in-electrum).
+
+### Prerequisites:
+-  Wallet private key
+
+### Getting a wallet private key
+
+### Using Liquality
+
+To get your wallet private key, follow the steps below; 
+
+Note: To set up a Liquality or a Metamask wallet. Check here for a step by step process for how to add the Rootstock network to the Liquality browser wallet.
+
+**Step 1**: Open the Liquality wallet on your browser, you can find these in the extensions tab in your browser.
+
+**Step 2**: Check that you’re connected to the Rootstock network
+
+![liquality - connection](/assets/img/guides/two-way-peg-app/liquality/connection.png)
+
+**Step 3**: Choose the asset you want to use for the transaction
+![liquality - assets](/assets/img/guides/two-way-peg-app/liquality/connection.png)
+
+**Step 4**: Click on the menu by the right, and this will open up a list of options. 
+
+**Step 5**: Choose the option “Export private key” in the menu
+
+![liquality - export_private_key](/assets/img/guides/two-way-peg-app/liquality/export_private_key.png)
+
+**Step 6**: Choose “I have privacy”
+
+![liquality - privacy](/assets/img/guides/two-way-peg-app/liquality/privacy.png)
+
+**Step 7**: Fill out wallet password and check “I understand the risk and have privacy”
+
+**Step 8**: Copy the private key and click on “Done”
+
+### Using Metamask
+
+**Step 1**: Open the Metamask wallet on your browser, you can find this in the extensions tab in your browser.
+
+**Step 2**: Click on the menu icon by the right
+
+**Step 3**: Choose “Account details”
+
+![metamask - privacy](/assets/img/guides/two-way-peg-app/metamask/account_details.png)
+
+**Step 4**: Then click on the “Export private key” button
+
+![metamask - export_private_key](/assets/img/guides/two-way-peg-app/metamask/export_private_key.png)
+
+**Step 5**: Fill out wallet password and click on “Confirm”
+
+![metamask - fill_wallet_password](/assets/img/guides/two-way-peg-app/metamask/fill_wallet_password.png)
+
+**Step 6**: Copy the private key and click on “Done”
+
+![metamask - copy_assets](/assets/img/guides/two-way-peg-app/metamask/copy_assets.png)
+
+### Converting BTC to RBTC
+
+Before converting the funds, we need to convert the private key into a [Wallet Import Format](https://learnmeabitcoin.com/technical/wif). 
+
+To do this, follow the steps below;
```

Comment:
I suggest changing this to reflect that we recommend using a standalone approach rather than using the website to parse the WIF.
We should even put a message discouraging users from using learnmeabitcoin...

Reply:
Ok. Noted

Reply:
Alex just answered this

Reply:
Thanks @owans .